tests/Feature/Student/CollaborationControllerTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Api;
+
+use Tests\TestCase;
+use App\Models\Student;
+use App\Models\Resume;
+use App\Models\Collaboration;
+use App\Service\CollaborationService;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Exception;
+
+class CollaborationControllerTest extends TestCase
+{
+    /** @test */
+    public function it_returns_collaboration_details_for_valid_uuid()
+    {
+
+        $student = Student::factory()->create();
+        $resume = Resume::factory()->create(['student_id' => $student->id]);
```

Comment:
The two previous instructions generate one resume and two students, and I don't think it's really necessary.

As a suggestion, replace:
~~~
$resume = Resume::factory()->create(['student_id' => $student->id]);
~~~
with:
~~~
$resume = $student->resume()->create();
~~~